storybook/stories/CustomerAppeal/CustomerAppealScreen.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { View } from "react-native";
+import { CustomerAppealScreen } from "../../../src/components/CustomerAppeal/CustomerAppealScreen";
+import { CampaignPolicy } from "../../../src/types";
+import { navigation, mockReactNavigationDecorator } from "../mocks/navigation";
+import { provider as MockProvider } from "../mocks/provider";
+
+const products: CampaignPolicy[] = [
+  {
+    category: "Lost token",
+    categoryType: "APPEAL",
+    name: "Lost token",
+    alert: {
+      label: "*chargeable",
```

Comment:
I can't seem to see the `*chargeable` label, is this intended?

Reply:
in order to show. need to mock  data.  the mock of API service will be handle it different PR.